internal/models/linking.go

```diff
@@ -3,6 +3,8 @@ package models
 import (
 	"strings"
 
+	p "github.com/supabase/gotrue/internal/api/provider"
```

Comment:
Why a single letter? Can we refactor any other `provider` so this remains normal?